api/admin/controller/search_service_self_tests.py

```diff
@@ -7,7 +7,7 @@
     ExternalIntegration
 )
 from core.external_search import ExternalSearchIndex
-from core.tests.test_external_search import ExternalSearchTest
```

Comment:
This creates a reference to the test function which doesn't seem necessary and breaks the build process. `core/tests/test_external_search` calls `core/testing` so nothing has changed.